src/patterns/Header/Header.vue

```diff
@@ -1,17 +1,60 @@
 <template>
-  <header class="vocab header">
-    <!-- TODO -->
-  </header>
+  <header>
+      <nav class="navbar" :class = size>
```

Comment:
Your indentation is still off here. Please use an editor like VS Code or an IDE like WebStorm to avoid issues like this.